example/extensions/lib_custom_op/transposecsr_lib.cc

```diff
@@ -116,6 +116,8 @@ MXReturnValue inferType(std::map<std::string, std::string> attrs,
 MXReturnValue inferSType(std::map<std::string, std::string> attrs,
                         std::vector<int> &instypes,
                         std::vector<int> &outstypes) {
+  if (instypes[0] != kCSRStorage)
+    return MX_FAIL;
```

Comment:
error message telling users they provided the wrong storage type?

Reply:
Added.